rest/src/main/kotlin/service/InteractionService.kt

```diff
@@ -183,10 +186,51 @@ class InteractionService(requestHandler: RequestHandler) : RestService(requestHa
         }
 
 
-    suspend fun getGuildCommand(applicationId: Snowflake, guildId: Snowflake, commandId: Snowflake) =
-        call(Route.GuildApplicationCommandGet) {
-            keys[Route.ApplicationId] = applicationId
-            keys[Route.GuildId] = guildId
-            keys[Route.CommandId] = commandId
-        }
+    suspend fun getGuildCommand(applicationId: Snowflake, guildId: Snowflake, commandId: Snowflake) = call(Route.GuildApplicationCommandGet){
+        keys[Route.ApplicationId] = applicationId
+        keys[Route.GuildId] = guildId
+        keys[Route.CommandId] = commandId
+    }
+
+    suspend fun getGuildApplicationCommandPermissions(
+            applicationId: Snowflake,
+            guildId: Snowflake,
+    ): DiscordGuildApplicationCommandPermissions = call(Route.GuildApplicationCommandPermissionsGet) {
+        keys[Route.ApplicationId] = applicationId
+        keys[Route.GuildId] = guildId
+    }
+
+    suspend fun getApplicationCommandPermissions(
+            applicationId: Snowflake,
+            guildId: Snowflake,
+            commandId: Snowflake,
+    ): DiscordGuildApplicationCommandPermissions = call(Route.ApplicationCommandPermissionsGet) {
+        keys[Route.ApplicationId] = applicationId
+        keys[Route.GuildId] = guildId
+        keys[Route.CommandId] = commandId
+    }
+
+    suspend fun editApplicationCommandPermissions(
+            applicationId: Snowflake,
+            guildId: Snowflake,
+            commandId: Snowflake,
+            request: ApplicationCommandPermissionsEditRequest,
+    ) = call(Route.ApplicationCommandPermissionsPut) {
+        keys[Route.ApplicationId] = applicationId
+        keys[Route.GuildId] = guildId
+        keys[Route.CommandId] = commandId
+
+        body(ApplicationCommandPermissionsEditRequest.serializer(), request)
+    }
+
+    suspend fun bulkEditApplicationCommandPermissions(
+        applicationId: Snowflake,
+        guildId: Snowflake,
+        request: List<PartialDiscordGuildApplicationCommandPermissions>,
+    ) = call(Route.ApplicationCommandPermissionsBatchPut) {
+        keys[Route.ApplicationId] = applicationId
+        keys[Route.GuildId] = guildId
+
+        body(serializer(), request)
```

Comment:
`serializer()`? use `ListSerializer(DiscordPartialGuildApplicationCommandPermission.serializer())`
Also Discord should be prefixed in the name

Reply:
Oh that's how you do that